components/Projects.js

```diff
@@ -9,41 +9,39 @@ export default function Projects() {
           Projects
         </h1>
       </div>
-      {/* Grid starts here */}
-      <div className="bg-[#F1F1F1] dark:bg-gray-900">
-        <div className="max-w-6xl mx-auto grid grid-cols-1 md:grid-cols-2 gap-8 py-20 pb-40">
-          {userData.projects.map((proj, idx) => (
-            <ProjectCard
-              title={proj.title}
-              link={proj.link}
-              imgUrl={proj.imgUrl}
-              number={`${idx + 1}`}
-            />
-          ))}
-        </div>
+      <div className="bg-[#F1F1F1] dark:bg-gray-900 p-10 grid grid-cols-1 md:grid-cols-2">
+        {userData.projects.map((project, index) => (
+          <ProjectCard projectID={index} />
+        ))}
       </div>
     </section>
   );
 }
 
-const ProjectCard = ({ title, link, imgUrl, number }) => {
+const ProjectCard = ({ projectID }) => {
+  const project = userData.projects[projectID];
```

Comment:
Why not just pass the `project` as a prop on line 14? Then you don't have to go dig back into the array